pkg/appgw/configbuilder.go

```diff
@@ -48,19 +48,22 @@ func NewConfigBuilder(context *k8scontext.Context, appGwIdentifier *Identifier,
 
 // Build gets a pointer to updated ApplicationGatewayPropertiesFormat.
 func (c *appGwConfigBuilder) Build(cbCtx *ConfigBuilderContext) (*n.ApplicationGateway, error) {
+	glog.V(5).Infof("-----Generating Probes-----")
```

Comment:
I generally do not find these useful.  I'd **expect** AGIC to do this. It'd be certainly more interesting if for some reason it was **not** doing this - then I'd like to know.  I suspect these would add to the already very noisy log.